MoneyDataType/SystemCurrency.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace OrchardCore.Commerce.Money
+{
+    public enum SystemCurrency
```

Comment:
Introduced so Currencies can be requested in a similar way that colors can.  I use this a lot in my applications.  It's really useful.

Reply:
The static properties that give you directly the currency object seem to be richer. Also, no reason to single out those. Finally, the naming should be something like `CurrencyIsoSymbol` rather than `SystemCurrency`: they're symbols, not currencies, and they're not really system.

Reply:
System Currencies removed.